ui/job-view/headerbars/NotificationsMenu.jsx

```diff
@@ -25,6 +26,8 @@ class NotificationsMenu extends React.Component {
         return faInfoCircle;
       case 'success':
         return faCheck;
+      default:
+        return null;
```

Comment:
Also, I wasn't sure what to write in some default cases, so I just returned null.
Is this okay?

Reply:
Change the switch to an object:
```javascript
const severityToIcon = {
  danger: faBan,
};
return severityToIcon[severity];
```
You're right that it is unclear how to set a default.

Reply:
Yes, this is fine.